README.md

```diff
@@ -1,159 +1,173 @@
 
 ## Contributing
 
 We are excited to work alongside you, our amazing community, to build and enhance Windows Terminal\!
 
-We ask that **before you start work on a feature that you would like to contribute**, please read our [Contributor's Guide](https://github.com/microsoft/terminal/blob/master/doc/contributing.md). We will be happy to work with you to figure out the best approach, provide guidance and mentorship throughout feature development, and help avoid any wasted or duplicate effort.
-
-> 👉 **Remember\!** Your contributions may be incorporated into future versions of Windows\! Because of this, all pull requests will be subject to the same level of scrutiny for quality, coding standards, performance, globalization, accessibility, and compatibility as those of our internal contributors.
-
-> ⚠ **Note**: The Command-Line Team is actively working out of this repository and will be periodically re-structuring the code to make it easier to comprehend, navigate, build, test, and contribute to, so **DO expect significant changes to code layout on a regular basis**.
-
-## Documentation
-
-All documentation is located in the `./doc` folder. If you would like to contribute to the documentation, please submit a pull request.
+***BEFORE you start work on a feature/fix***, please read & follow our [Contributor's Guide](https://github.com/microsoft/terminal/blob/master/doc/contributing.md) to help avoid any wasted or duplicate effort.
 
 ## Communicating with the Team
 
-The easiest way to communicate with the team is via GitHub issues. Please file new issues, feature requests and suggestions, but **DO search for similar open/closed pre-existing issues before you do**.
+The easiest way to communicate with the team is via GitHub issues.
 
-Please help us keep this repository clean, inclusive, and fun\! We will not tolerate any abusive, rude, disrespectful or inappropriate behavior. Read our [Code of Conduct](https://opensource.microsoft.com/codeofconduct/) for more details.
+Please file new issues, feature requests and suggestions, but **DO search for similar open/closed pre-existing issues before creating a new issue.
 
 If you would like to ask a question that you feel doesn't warrant an issue (yet), please reach out to us via Twitter:
 
-  * Rich Turner, Program Manager: [@richturn\_ms](https://twitter.com/richturn_ms)
-
-  * Dustin Howett, Engineering Lead: [@dhowett](https://twitter.com/DHowett)
-  
-  * Michael Niksa, Senior Developer: [@michaelniksa](https://twitter.com/MichaelNiksa)
-
-  * Kayla Cinnamon, Program Manager (especially for UX issues): [@cinnamon\_msft](https://twitter.com/cinnamon_msft)
+* Kayla Cinnamon, Program Manager: [@cinnamon\_msft](https://twitter.com/cinnamon_msft)
+* Rich Turner, Program Manager: [@richturn\_ms](https://twitter.com/richturn_ms)
+* Dustin Howett, Engineering Lead: [@dhowett](https://twitter.com/DHowett)
+* Michael Niksa, Senior Developer: [@michaelniksa](https://twitter.com/MichaelNiksa)
```

Comment:
I would love to add Mike and Carlos here too. It's incomplete without them.

Reply:
Done, thanks.